tests/mock_requests.py

```diff
@@ -81,8 +77,8 @@ def mock_requests_get(*args, **kwargs):
         return MockingResponse(text, status_code, error_msg)
 
     def _save_collected_responses():
-        with open(os.path.join(_data_filepath), 'wb') as fd:
-            pickle.dump(_collected_responses, fd, protocol=2)
+        with open(os.path.join(_data_filepath), 'w') as fd:
+            json.dump(_collected_responses, fd)
```

Comment:
We might want to use `indent=2`  in the Json dump. This falls back to deciding whether the data is part of the test *documentation* and whether it is readable. On one side it would make the file slightly bigger, on the other it would better be diffed in git. If we chose this path, we probably want to sort the responses first.
```python
_collected_responses.sort()
json.dump(_collected_responses, fd, indent=2)
```
This is pretty minor, so I leave it to you to decide.

Reply:
agreed.